experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/OTLPMetricExporterOptions.ts

```diff
@@ -13,10 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { otlpTypes } from '@opentelemetry/exporter-trace-otlp-http';
+
 import { AggregationTemporality } from '@opentelemetry/sdk-metrics-base';
+import { OTLPExporterConfigBase } from '@opentelemetry/otlp-exporter-base';
 
-export interface OTLPMetricExporterOptions extends otlpTypes.OTLPExporterConfigBase {
+export interface OTLPMetricExporterOptions extends OTLPExporterConfigBase {
```

Comment:
Would it make sense to move `aggregationTemporality` to `OTLPExporterConfigBase` (or move `OTLPMetricExporterOptions` to `@opentelemetry/otlp-exporter-base`)? As it is, the grpc exporter will have a dependency on the http exporter only due to importing `OTLPMetricExporterOptions` and the default options.